pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -165,52 +176,49 @@ func (w *watcher) run(ctx context.Context, repo git.Repo, repoCfg config.PipedRe
 	}
 }
 
+// cloneRepo clones the git repository under the working directory.
+func (w *watcher) cloneRepo(ctx context.Context, repoCfg config.PipedRepository) (git.Repo, error) {
+	repo, err := w.gitClient.Clone(ctx, repoCfg.RepoID, repoCfg.Remote, repoCfg.Branch, filepath.Join(w.workingDir, repoCfg.RepoID))
```

Comment:
Maybe we should use a random directory instead of fixing it with the repository ID because this could be affected by the error at line 142.

Reply:
Right.